docs/ota-client-guide/modules/ROOT/pages/developer-tools.adoc

```diff
@@ -16,49 +16,49 @@ You can use it to easily evaluate OTA Connect functionality. By default, it's in
 ====
 __Related Tool__: *libaktualizr*
 
-This is the C++ library that powers aktualizr. Aktualizr is actually a thin wrapper around this library. 
+This is the C++ library that powers aktualizr. Aktualizr is actually a fairly thin wrapper around this library.
 
-You can use it to add OTA update functionality to your own client applications.
+You can use it to customize the software update process for your particular device or vehicle--for example, checking for necessary install conditions, implementing complex rollback logic, or integrating with other systems for flashing updates.
 
 Later in this guide, we show you xref:libaktualizr-why-use.adoc[how to use libaktualizr].
 ====
 
 | *Yocto* | Yocto is an open source toolkit that you can use to build Linux-based disk images for almost any embedded device. It uses the concept of "recipes" to build disk images for different hardware architectures.
 
-You'll be using the common Yocto command `bitbake` to build disk images. You can start with our demo recipes then make your own recipes.
+You'll be using the common Yocto command `bitbake` to build disk images. You can start with our demo recipes, then make your own recipes.
 
 [NOTE]
 ====
 __Related Tool__: *meta-updater*
 
-The "meta-updater" defines some of the important "ingredients" that go into our Yocto recipes. Yocto uses the concept of *layers* to bundle different types of "ingredients". In the case of OTA Connect, our "meta-updater" layer adds the aktualizr client and the OSTree tool to your Yocto builds.
+The "meta-updater" layer defines some of the important "ingredients" that go into our Yocto recipes. Yocto uses the concept of *layers* to bundle different types of "ingredients". In the case of OTA Connect, our "meta-updater" layer adds the aktualizr client and the OSTree tool to your Yocto builds.
```

Comment:
The quotes around "meta-updater" feel strange to me. That is the name, it's not a euphemism or nickname or something.

Reply:
Yeah, I think I agree. I'll push a quick fix.